tests/test_configure_feedstock.py

```diff
@@ -211,7 +211,13 @@ def test_py_matrix_on_azure(py_recipe, jinja_env):
     assert len(os.listdir(matrix_dir)) == 6
 
 
-def test_stdlib_on_azure(stdlib_recipe, jinja_env):
+def test_stdlib_on_azure(stdlib_recipe, jinja_env, request):
+    conda_build_param = request.node.callspec.params["config_yaml"]
+    if conda_build_param == "rattler-build":
+        pytest.skip(
+            "skipping test for rattler-build usecase as we currently we don't have stdlib"
+        )
```

Comment:
Is there a rattler-build issue we can link here?

Reply:
yes, let me add it